src/validate/throw.js

```diff
@@ -0,0 +1,9 @@
+'use babel'
+
+const throwIfFail = (msg, passed) => {
+  if (!passed) {
+    throw new Error(msg)
+  }
+}
+
+export default throwIfFail
```

Comment:
Just move the `export default` to the definition above. That way if more things are added later you just need to remove `default`.

Reply:
I don't think I follow you on "just need to remove `default`. As it is now, it already has a named identifier `throwIfFail`, so if we add more stuff and decide it shouldn't be the default, we can just put `export` in front of the function and move on. To assign it directly to `default`, we must remove the identifier, so if we change it later we must go through the naming process again. Or maybe I'm misunderstanding?

Also I think it is more self documenting this way. If we remove the identifer, we probably need to make the file name significantly more descriptive.

Reply:
Sounds great! Only problem is that's not valid Javascript. Lol 

> Error | ESLint | Parsing error: Unexpected token const (Fatal) | 3:16 | src\validate\throw.js

Reply:
Also, with current config, if if we try to export it both by name and as default
```
export const throwIfFail = /*... */
export default throwIfFail
```
then ESLint will yell at us later on if we try to import from default.
``` 
import throwIfFail from './validate/throw' // eslint error: import/no-named-as-default
```

Reply:
😆 Hmmm, I was sure I've used that before. I'm fine with leaving it as is for now.